__tests__/services/firebase/Authentication.test.js

```diff
@@ -12,14 +17,24 @@ let mockFetchProviders = jest.fn()
 jest.mock('firebase', () => ({
   auth: () => ({
     signInWithEmailAndPassword: () => Promise.resolve(mockFirebaseResponse),
+    createUserWithEmailAndPassword: jest.fn(() => Promise.resolve(mockFirebaseResponse)),
     fetchProvidersForEmail: mockFetchProviders
   })
 }))
 
 describe('Firebase authentication service', () => {
+  it('Should register new user and send a verification mail', async () => {
+    const email = 'new@user.com'
+    const password = 'pass123'
+    await signUp(email, password)
+    // expect(createUserWithEmailAndPassword).toHaveBeenCalledWith(email, password)
```

Comment:
The truth is: we should get the reference of `createUserWithEmailAndPassword` at line 20. But I have no idea how to do this. Looks like it's a simple change, but isn't. I let this commented for someone brave enough try to fix this. (In a fancy way, of course)